packages/manager/apps/web-office/public/translations/dashboard/users/change-password/Messages_fr_FR.json

```
@@ -1,19 +1,19 @@
 {
   "dashboard_users_change-password_title": "Modifier le mot de passe",
-  "dashboard_users_change-password_description": "Vous êtes sur le point de modifier le mot de passe de votre compte",
-  "dashboard_users_change-password_subdescription": "Voulez-vous continuez ?",
-  "dashboard_users_change-password_label1": "Mot de passe :",
-  "dashboard_users_change-password_label2": "Confirmation :",
-  "dashboard_users_change-password_label3": "Email :",
+  "dashboard_users_change-password_radio_1": "Je génère un mot de passe aléatoire",
+  "dashboard_users_change-password_radio_2": "Je personnalise mon mot de passe",
+  "dashboard_users_change-password_label_password": "Nouveau mot de passe* :",
```

```suggestion
  "dashboard_users_change-password_label_password": "Nouveau mot de passe",
```
I think non need ":" in label on other project we don't have and then the * we can put directly in code with conditional something 
```
mandatory ? '*' :  t('optional')
```